src/adapters/default/attendance.adapter.ts

```diff
@@ -116,16 +121,44 @@ export class AttendanceService {
   public async userSegment(attendance: string, date: string, request: any) {
     let axios = require("axios");
 
-    let data = {
+    let data: any = {
       filters: {
-        attendanceDate: {
-          eq: `${date}`,
-        },
         attendance: {
           eq: `${attendance}`,
         },
       },
     };
+    switch (date) {
```

Comment:
Please enable the below
- today
- yesterday
- thisweek
- lastweek
- thismonth
- lastmonth

Reply:
@coolbung sir done